SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -136,6 +136,8 @@
   private static final String AGENT_DETAIL_FORMAT = AGENTS_FORMAT + "/agent/%s/details";
   private static final String AGENTS_DECOMISSION_FORMAT =
     AGENTS_FORMAT + "/agent/%s/decommission";
+  private static final String EXTEND_AGENTS_DECOMISSION_FORMAT =
+    AGENTS_FORMAT + "/agent/%s/decommission/extend";
```

Comment:
Probably want to change language from `extend` to `update` since it's a more generic operation.